app/controllers/admin_controller.rb

```diff
@@ -1,12 +1,54 @@
-class AdminController < ApplicationController
-    layout "admin"
-    def index
-
-    end
-    
-    def room
-
-    end
-
-
-end
+class AdminController < ApplicationController
+  layout "admin"
+
+  def index
+    @rooms = Room.all
+  end
+    
+  def room
+    @rooms = Room.all
+  end
+
+  def add_room

```

Comment:
bu methodlar RoomController'a taşınmalı. Şimdilik en sona bırakabiliriz.
Sonradan bu yapacağımızı hatırlamak için not düşebilirsin bu sayfaya.
# TODO: move room methods to room controller

Bunun gibi todolar yazarsan boş zamanlarında hatalarını düzeltebilirsin.

Reply:
Evet bu konuyu konuşmuştuk master. O yüzden öncelikli bir konu olmadığından bekletiyorum.